codegenerator/cli/src/cli_args/interactive_init/mod.rs

```diff
@@ -149,10 +159,47 @@ pub async fn prompt_missing_init_args(
         .await
         .context("Failed getting template")?;
 
+    let api_token: Option<String> = match init_args.api_token {
+        Some(k) => Ok::<_, anyhow::Error>(Some(k)),
+        None if ecosystem.uses_hypersync() => {
+            let select = Select::new(
+                "Add an API token for HyperSync to your .env file?",
```

Comment:
I think we can omit mentioning the .env file, since it's an implementation detail and shouldn't really concern the user at the point.

Reply:
Hmmm, although that's exactly what this is doing right? IMO I like that it makes it clear that all it's doing is helping you add a token to your .env file and you can manually do it later if you want.